test/controllers/customers_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class CustomersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @customer = customers(:one)
+  end
+
+  test "should get index" do
+    get customers_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_customer_url
+    assert_response :success
+  end
+
+  test "should create customer" do
+    assert_difference('Customer.count') do
+      post customers_url, params: { customer: { email: @customer.email, first_name: @customer.first_name, last_name: @customer.last_name, phone: @customer.phone } }
```

Comment:
Sometimes it can get very hard to understand lines of code when they are very long. This can be a good alternative.
<img width="295" alt="Screen Shot 2021-06-18 at 12 03 37 PM" src="https://user-images.githubusercontent.com/47392623/122606077-3965a300-d02d-11eb-9039-0cd01b938331.png">

}`